### PR TITLE
Template update for nf-core/tools version 1.8

### PR DESCRIPTION
**Pull request by christopher-mohr**
Many thanks for contributing to nf-core/epitopeprediction!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/epitopeprediction branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/epitopeprediction)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by christopher-mohr** [comment]
We still have to sort out a couple of things here...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        run: |
+          wget -qO- get.nextflow.io | bash
+          sudo mv nextflow /usr/local/bin/
+      - name: Pull docker image
+        run: |
+          docker pull nfcore/epitopeprediction:dev && docker tag nfcore/epitopeprediction:dev nfcore/epitopeprediction:dev
+      - name: Run pipeline with test data
+        run: |
+          # TODO nf-core: You can customise CI pipeline run tests as required
+          # (eg. adding multiple test runs with different parameters)
```
Comment (ggabernet) [inline .github/workflows/ci.yml]: add the pipeline tests that you had with Travis here

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          nextflow run ${GITHUB_WORKSPACE} -profile test,docker
```
Comment (ggabernet) [inline .github/workflows/ci.yml]:
```suggestion
          nextflow run ${GITHUB_WORKSPACE} -profile test,docker -ansi-log false
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 - [date]
```
Comment (ggabernet) [inline CHANGELOG.md]:
```suggestion
## v1.0.0dev - [date]
```

File: Dockerfile [hunk 4]
```diff
@@ -7,7 +7,7 @@ COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
-ENV PATH /opt/conda/envs/nf-core-epitopeprediction-1.0dev/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-epitopeprediction-1.0.0/bin:$PATH
```
Comment (ggabernet) [inline Dockerfile]:
```suggestion
ENV PATH /opt/conda/envs/nf-core-epitopeprediction-1.0.0dev/bin:$PATH
```

File: Dockerfile [hunk 5]
```diff
@@ -7,7 +7,7 @@ COPY environment.yml /
...
 
 # Dump the details of the installed packages to a file for posterity
-RUN conda env export --name nf-core-epitopeprediction-1.0dev > nf-core-epitopeprediction-1.0dev.yml
+RUN conda env export --name nf-core-epitopeprediction-1.0.0 > nf-core-epitopeprediction-1.0.0.yml
```
Comment (ggabernet) [inline Dockerfile]: You don't need this line as this is already included in environment.yml

File: environment.yml [hunk 6]
```diff
@@ -1,11 +1,12 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-epitopeprediction-1.0dev
+name: nf-core-epitopeprediction-1.0.0
```
Comment (ggabernet) [inline environment.yml]:
```suggestion
name: nf-core-epitopeprediction-1.0.0dev
```

File: environment.yml [hunk 7]
```diff
@@ -1,11 +1,12 @@
...
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
+  - conda-forge::python=3.7.3
```
Comment (ggabernet) [inline environment.yml]: might be incompatible with your current python 2 requirements

**Comment by ggabernet** [comment]
something is really wrong with this PR, the environment.yml on BASE, is different from nf-core/dev. Let's close and start from scratch in a fork

**Pull request closed without merging**